sa/sa_test.go

```diff
@@ -769,6 +770,7 @@ func TestAddIssuedNames(t *testing.T) {
 func TestDeactivateAuthorization(t *testing.T) {
 	sa, _, cleanUp := initSA(t)
 	defer cleanUp()
+	features.Set(map[string]bool{"AllowAccountDeactivation": true})
```

Comment:
Why does `TestDeactivateAuthorization` need the `AllowAccountDeactivation` feature to be `true`? Similarly, why doesn't `TestDeactivateAccount` need to flip the feature flag? Did this `features.Set` perhaps get put in the wrong place?